config/rbac/hive_frontend_role_binding.yaml

```diff
@@ -7,6 +7,8 @@ roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
   name: hive-frontend
+groupNames:
```

Comment:
This is not a valid field for ClusterRoleBinding in rbac.authorization.k8s.io/v1.

Reply:
This is a field from ClusterRoleBinding in authorization.openshift.io/v1, which is not what is being used here. Even in that, the field is legacy and has been superseded by the use of a Group subject, as was done in #715.

Reply:
submitted a fix in #719